ui-components/src/locales/ar.json

```diff
@@ -1361,22 +1362,22 @@
         "independentLivingHousing": "স্বাধীন জীবিত সম্প্রদায়",
         "minAmiPercentageLabel": "الوحدات المخصصة لذوي الدخل",
         "minAmiPercentageOptions": {
-          "amiOption20": "20٪ من AMI أو أكثر",
-          "amiOption25": "25٪ من AMI أو أكثر",
-          "amiOption30": "30٪ من AMI أو أكثر",
-          "amiOption35": "35٪ من AMI أو أكثر",
-          "amiOption40": "40٪ من AMI أو أكثر",
-          "amiOption45": "45٪ من AMI أو أكثر",
-          "amiOption50": "50٪ من AMI أو أكثر",
-          "amiOption55": "55٪ من AMI أو أكثر",
-          "amiOption60": "60٪ من AMI أو أكثر",
-          "amiOption70": "70٪ من AMI أو أكثر",
-          "amiOption80": "80٪ من AMI أو أكثر",
-          "amiOption100": "100٪ من AMI أو أكثر",
-          "amiOption120": "120٪ من AMI أو أكثر",
-          "amiOption125": "125٪ من AMI أو أكثر",
-          "amiOption140": "140٪ من AMI أو أكثر",
-          "amiOption150": "150٪ من AMI أو أكثر"
+            "amiOption20": "20٪ من AMI أو أكثر",
```

Comment:
Hm, I worry that mixing formatting fixes with bug fixes might make merges more difficult. I'll defer to Will to comment though.

Reply:
I'm pretty sure that all the format changes here are for strings that don't exist in the upstream repo. But if we think this will complicate merges I can move the format changes into a separate PR

Reply:
We don't really merge the translation strings (other than additions) because ours are so far out of line with upstream. Upstream uses es/vi/zh, we use es/ar/bn, so there's not too much overlap. I think these are currently excluded from Prettier formatting, so if you really want to get it all sorted, do a PR just for formatting all the *.json and un-ignore them from Prettier so they stay formatted correctly.

Reply:
Reverted the formatting changes.